static/app/components/feedback/feedbackItem/feedbackItem.tsx

```diff
@@ -122,6 +133,49 @@ export default function FeedbackItem({feedbackItem, eventData, tags}: Props) {
   );
 }
 
+function TraceDataSection({
+  eventData,
+  crashReportId,
+  hasProject,
+}: {
+  crashReportId: string | undefined;
+  eventData: Event;
+  hasProject: boolean;
+}) {
+  // If there's a linked error from a crash report and only one other issue, showing both could be redundant.
+  // TODO: we could add a jest test .spec for this ^
+  const {oneOtherIssueEvent, traceEvents, isLoading, isError} = useTraceTimelineEvents({
+    event: eventData,
+  });
+  const hide: boolean =
+    isLoading ||
+    isError ||
+    traceEvents.length <= 1 || // traceEvents include the current event.
+    (hasProject && !!crashReportId && oneOtherIssueEvent?.id === crashReportId);
+
+  const [sentMetric, setSentMetric] = useState(false);
+  const ffEnabled = true; // TODO:
+
+  if (!isLoading && !isError && !sentMetric) {
+    if (isError) {
+      metrics.increment('feedback.trace_section.error');
+    } else {
+      metrics.increment('feedback.trace_section.events_loaded', traceEvents.length - 1);
+      if (hasProject && !!crashReportId && oneOtherIssueEvent?.id === crashReportId) {
+        metrics.increment('feedback.trace_section.crash_report_dup');
+      }
+    }
+    setSentMetric(true);
+  }
```

Comment:
This should be put into a `useEffect(() => { ... }, [])` so it only runs once on mount.

Reply:
oh yes!

Reply:
I can prob remove the sentMetric flag then?

Reply:
Actually think we still need it since the useEffect has a lot of dependencies

Reply:
ya, useEffect will replace all that.

Reply:
> Actually think we still need it since the useEffect has a lot of dependencies

the dependencies should be ok unless the data refreshes very often. otherwise they should be stable each time this renders which means that the effect will only run once when new values appear